### DISCUSSION TITLE
Reveal Skd for WPF and .Net 8

### DISCUSSION THREAD

**Discussion by fzandomeni** [RevealBi/Reveal.Sdk#336] [WPF]
Does Reveal SDK for WPF have an official timeline for .NET 8 support?
Is there any workaround for embedding a WPF Reveal dashboard into a WPF application compiled in .NET 8?

**Comment by brianlagunas** [comment]
Currently, Reveal SDK for WPF supports .NET Framework. For .NET 8, no specific plans or timelines have been finalized.

One potential workaround would be to use our web-based version of Reveal and embed it within your WPF application using a web view control. This approach allows you to leverage the full capabilities of Reveal while running on .NET 8.

**Comment by fzandomeni** [comment]
That sounds intriguing. However, I haven't worked with the Reveal web
version before.
Do you know of any articles or resources that could help with transitioning
from the WPF SDK to the web?
Any recommendations would be greatly appreciated. Thanks!

**Reply by brianlagunas** [reply]
Here is the documentation for the Reveal SDK for Web: https://help.revealbi.io/web/

The idea is to wrap the javascript client in a webview2.
